jsk_perception/node_scripts/split_image.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+import cv_bridge
+import rospy
+from sensor_msgs.msg import Image
+from jsk_topic_tools import ConnectionBasedTransport
+
+
+class SplitImage(ConnectionBasedTransport):
+
+    def __init__(self):
+        super(SplitImage, self).__init__()
+        self.vertical_parts = rospy.get_param('~vertical_parts', 1)
+        self.horizontal_parts = rospy.get_param('~horizontal_parts', 1)
+        self.pubs = []
+        for v in range(self.vertical_parts):
+            pubs = []
+            for h in range(self.horizontal_parts):
+                pubs.append(
+                    self.advertise('~output/{}/{}'.format(v, h),
```

Comment:
can you add string and add zero padding?

```suggestion
                    self.advertise('~output/vertical{0:02}/horizontal{1:02}'.format(v, h),
```

Reply:
Thank you for your advice.
I updated.